source/Angle.h

```diff
@@ -50,6 +50,7 @@ class Angle {
 	Point Unit() const;
 	// Convert an Angle object to degrees, in the range -180 to 180.
 	double Degrees() const;
+	double Degrees360() const;
```

Comment:
why do we need to convert it this way?

Reply:
I am not really sure about this one... Just copied the way the Degrees() function was done.